### DISCUSSION TITLE
How can i distable ctrl+v AS PASTE SHORT-KEY and use vim default action

### DISCUSSION THREAD

**Discussion by DemingQiao** [LazyVim/LazyVim#769] [Q&A] [answered]
How can i distable ctrl+v AS PASTE SHORT-KEY and use vim default action

**Comment by folke** [comment, marked as answer]
`<c-v>` is not remapped in LazyVim and is the default. (see `:nmap <c-v>`)

If it behaves differently, then it must be your terminal that has an action bound to that key.

**Reply by DemingQiao** [reply]
Thanks a lot!

**Comment by linsui** [comment]
I thought it behaves differently because of [this option](
https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/config/options.lua#L8). Since the clipboard is synced with the register, when `<c-v>` on selection the clipboard is also changed.
